.gitignore

```diff
@@ -10,3 +10,6 @@ gradlew.bat
 # Ignore IntelliJ project files
 .idea
 gcp-variant-transforms-java.iml
+
+#Ignore VSCode files
+.vscode/
```

Comment:
Wasn't this added in previous PRs? Doesn't matter much, but maybe some commit mishap.

Reply:
I'm not sure if this commit still affects this PR. Maybe we can take it out